src/domain/hanke/edit/HankeForm0.tsx

```diff
@@ -30,7 +29,6 @@ const Form0: React.FC<FormProps> = ({ errors, register, formData }) => {
 
   return (
     <div className="form0">
-      <EditDisabledNotification formData={formData} />
```

Comment:
There doesn't seem to be other uses for the `EditDisabledNotification` component, can it be removed as well?

Reply:
EditDisabledNotification component also removed as it is not used anymore.